src/components/Header/Header.js

```diff
@@ -15,6 +15,6 @@ const Header = () => (
     </h1>
   </a>
   </header>
-)
+);
```

Comment:
This project uses eslint and prettier for style formatting. Having no semi-colons is part of the linter rules (part of the https://standardjs.com style). So all semi-colons should be removed.